tests/infra/e2e_args.py

```diff
@@ -246,7 +246,8 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
     parser.add_argument(
         "--snapshot-tx-interval",
         help="Number of transactions between two snapshots",
-        default=None,
+        type=int,
+        default=10,
```

Comment:
Why 10?

Reply:
As explained above, we want snapshots to be generated frequently enough that nodes automatically join from snapshots whenever possible.

Reply:
Got it.